### DISCUSSION TITLE
Haskell-tools not appearing in nvim after installing using luarocks.

### DISCUSSION THREAD

**Discussion by shavet-akiya** [mrcjkb/haskell-tools.nvim#389] [Q&A]
Not sure whats happening, healthcheck doesn't return anything, it just doesnt exist apparently

**Comment by mrcjkb** [comment]
hey 👋 

I'm afraid I can't help without more context.
I know that the luarocks package works (I've used it before with rocks.nvim).
So it's likely an issue with how you are telling Neovim to search for luarocks packages.

**Reply by shavet-akiya** [reply]
okay so i just uninstalled everything and reinstalled, and it worked. is there functionality for a tab autocomplete? i was reading through the config stuff but im not sure if i missed it, im quite inexperienced with nvim configs

**Reply by mrcjkb** [reply]
This plugin doesn't provide autocompletion.
You can enable Neovim's built-in autocompletion (`:h vim.lsp.completion`)
or use a plugin like [care.nvim](https://max397574.github.io/care.nvim/) or [nvim-cmp](https://github.com/hrsh7th/nvim-cmp).
